component/component/rbac_e2e.jsonnet

```diff
@@ -4,39 +4,54 @@ local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 local params = inv.parameters.appcat;
 
-local e2eNs = kube.Namespace('appcat-e2e') + {
+local e2eNs = kube.Namespace('schedar-e2e') + {
   metadata+: {
     labels+: {
-      'appuio.io/organization': 'vshn-e2e-tests',
+      'appuio.io/organization': 'vshn',
     },
   },
 };
 
-local e2eSA = kube.ServiceAccount('appcat-e2e') + {
+local e2eSA = kube.ServiceAccount('github-ci') + {
   metadata+: {
-    namespace: 'appcat-e2e',
+    namespace: 'schedar-e2e',
   },
 };
 
-local e2eRoleBinding = kube.RoleBinding('appcat-e2e') + {
+local e2eRoleBinding = kube.RoleBinding('schedar-e2e') + {
   metadata+: {
-    namespace: 'appcat-e2e',
+    namespace: 'schedar-e2e',
   },
-  roleRef_: kube.ClusterRole('admin'),
+  roleRef_: kube.ClusterRole('gitlab-ci'),
```

Comment:
github-ci

Reply:
I just realized that they both don't exist... I guess I just use `admin` here. It's also what the docs APPUiO use.